src/components/Stream/assistant.tsx

```diff
@@ -57,7 +58,7 @@ const Assistant = ({ message, itemId }: AssistantProps) => {
             <Media
               e2e="assistant-iframe"
               key={uuidV4()}
-              title={it[it.type]?.title||''}
+              title={it[it.type]?.title || ''}
               background={`url(https://img.youtube.com/vi/${extractVideoCode(it[it.type]?.url)}/hqdefault.jpg)`}
               url={extractVideoCode(it[it.type]?.url || 'https://www.youtube.com/embed/g4B8Dhl4pxY')}
               type={Definition.video}
```

Comment:
@IlievaDayana I don't see any fallback for the background similar to the one for `url` attribute. Can you provide something?